Dockerfile

```diff
@@ -26,4 +26,4 @@ COPY . .
 
 RUN bundle exec rake assets:precompile
 
-CMD bundle exec rails db:migrate && bundle exec rails server -b 0.0.0.0
+CMD bundle exec rails db:migrate && db:seed && bundle exec rails server -b 0.0.0.0
```

Comment:
this `CMD` gets overridden during deployment
 
https://github.com/DFE-Digital/register-trainee-teachers/blob/master/terraform/modules/paas/variables.tf#L40

Reply:
I've reverted this and kept it out https://github.com/DFE-Digital/register-trainee-teachers/pull/136/commits/af428fb38391e196772f5f221ab5f6cdbeb25cef